scripts/l1_node.sh

```diff
@@ -1,18 +1,21 @@
 #!/bin/bash
-set -exu
+# set -exu
```

Comment:
```
       -x    The shell shall write to standard error a trace for each command after it expands the command and before it executes it. It is unspecified whether the command that turns tracing off is traced.
```

i'm fine with deleting, but lets delete this commented out line

Reply:
not sure where this comment is gotten from? not in this code

Reply:
it's present in the diff, you commented out line 2?

Reply:
yeah I mean the "The shell shall write to standard error a trace for each command after it expands ..."

Reply:
I commented it out just for convience for whoever want to debug and see what variable is what

Reply:
removed!